browser/resources/settings/brave_rewards_page/brave_rewards_page.js

```diff
@@ -164,7 +164,7 @@ class SettingsBraveRewardsPage extends SettingsBraveRewardsPageBase {
   ready() {
     super.ready()
     this.openRewardsPanel_ = () => {
-      chrome.braveRewards.openBrowserActionUI('brave_rewards_panel.html')
+      chrome.braveRewards.openRewardsPanel()
```

Comment:
Would we change these to be a mojom / pagehandler API in the future and remove the rewards extension API fully?

Reply:
Oh I see greaselion uses it too, so we'd need that until that backend gets converted to something more native.

Reply:
Right - the idea is to leave the `chrome.braveRewards` API in place at least until we remove the extension, and then start transitioning away from it.